Cubical/Algebra/CommRing/Localisation/InvertingElements.agda

```diff
@@ -87,7 +87,7 @@ module InvertingElementsBase (R' : CommRing ℓ) where
                                λ _ → eq/ _ _ ((0r , ∣ 1 , sym (·IdR 0r) ∣₁) , useSolver _ _)
   where
   useSolver : ∀ s r → 0r · 1r · s ≡ 0r · r · 0r
-  useSolver = solve R'
```

Comment:
Would have expected this to be inlineable as well.

Reply:
This one is also mysterious - I put it in #1097

Reply:
Also solved now